m8-games/ship-adventure/Game1.cs

```diff
@@ -47,6 +54,14 @@ protected override void LoadContent()
 
             Font = Content.Load<SpriteFont>("defaultFont");
 
+            shootingSound = Content.Load<SoundEffect>("laser-cannon-chopped");
+            soundFxInstance = shootingSound.CreateInstance();
+            soundFxInstance.Volume = .3f;
+
+            mainTheme = Content.Load<Song>("battleThemeA");
+            MediaPlayer.Volume = .2f;
+            MediaPlayer.Play(mainTheme);
+
             ShotOffset = new Vector2(Ship.Width / 2, Ship.Height / 2);
```

Comment:
so. using new in. the game loop is going to be a big no no down the line. If you think about it you basically are allocating new memory every single time you run your main loop, and then de-allocating it. That's great for apps but bad for games.

It's kind of common to make one vector, and then re-use. it, we'll have a lot of methods like cleanup or something ilke that